app/webpacker/components/NewcomerChecks/NameFormatChecker.jsx

```diff
@@ -1,32 +1,45 @@
 import React from 'react';
 import { useQuery } from '@tanstack/react-query';
-import { List, Message } from 'semantic-ui-react';
+import { Button, Message, Table } from 'semantic-ui-react';
 import newcomerNameFormatCheck from './api/newcomerNameFormatCheck';
 import Loading from '../Requests/Loading';
 import Errored from '../Requests/Errored';
 import I18n from '../../lib/i18n';
 
-export default function NameFormatChecker({ competitionId }) {
+export default function NameFormatChecker({ competitionId, setUserIdToEdit }) {
   const {
-    data: nameFormatChecks, isLoading, isError, error,
+    data: nameFormatChecks, isFetching, isError, error,
   } = useQuery({
     queryKey: ['newcomer-name-format-checks', competitionId],
     queryFn: () => newcomerNameFormatCheck({ competitionId }),
   });
 
-  if (isLoading) return <Loading />;
+  if (isFetching) return <Loading />;
   if (isError) return <Errored error={error} />;
   if (nameFormatChecks.length === 0) {
     return <Message positive>All names looks good. Thanks for checking.</Message>;
   }
 
   return (
-    <List celled>
-      {nameFormatChecks.map(({ id, args }) => (
-        <List.Item>
-          {I18n.t(`validators.persons.${id}`, args)}
-        </List.Item>
-      ))}
-    </List>
+    <Table celled>
+      <Table.Header>
+        <Table.Row>
+          <Table.HeaderCell>Warning message</Table.HeaderCell>
+          <Table.HeaderCell>Action</Table.HeaderCell>
+        </Table.Row>
+      </Table.Header>
+      <Table.Body>
+        {nameFormatChecks.map(({ id, issues }) => issues.map(({ id: issueId, args }) => (
```

Comment:
I still don't understand your argument. Yes, the format has changed from before, I get that. But why does that mean that a "map inside map" is okay?

I am mostly confused because this line returns an array *of an array* of React nodes. I am genuinely surprised that this even works without crashing.